setup.py

```diff
@@ -8,14 +8,15 @@
 # NOTE: These are tested in `continuous_integration/travis/test_imports.sh` If
 # you modify these, make sure to change the corresponding line there.
 extras_require = {
-  'array': ['numpy >= 1.13.0', 'toolz >= 0.7.3'],
+  'array': ['numpy >= 1.13.0', 'toolz >= 0.7.3', 'psutil >= 0.5'],
```

Comment:
I'd be more comfortable leaving `psutil` as an optional dependency for `'array'`, `'bag'`, `'dataframe'`, and `'delayed'`. It's already listed in our table of optional dependencies https://docs.dask.org/en/latest/install.html#optional-dependencies. Would you mind removing those requirements here?

Reply:
done